index.js

```diff
@@ -1,17 +1,38 @@
 const express = require("express");
-const cors = require("cors");
-const bcrypt = require("bcryptjs");
 const session = require("express-session");
 const db = require("./database/dbConfig.js");
 const KnexSessionStore = require("connect-session-knex")(session);
 
-//--------------------------------------//
+const path = require("path");
+
+const server = express();
+
+const login = require("./Login/login.js");
+const configMiddleware = require("./config/middleware.js");
+
+//configure middleware for the server
+configMiddleware(server);
+
+server.get("/download", (req, res, next) => {
```

Comment:
Still not exactly sure what this endpoint is doing.

Reply:
The /download is used to connect the routers with the configuration files and index.js. It was also something referenced by Luis in the videos a while back. Same with the error handler.